app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  after_save :update_likes_counter
+
+  def update_likes_counter
+    post.update(likes_counter: post.likes.count)
+  end
```

Comment:
- [ ] OPTIONAL: I would recommend to increment counter every time by 1, to do that you can simply use :
```ruby
def update_comments_counter
  post.increment!(:comments_counter)
end
```